Vagrantfile

```diff
@@ -56,16 +76,30 @@ Vagrant.configure(2) do |config|
     end
   end
 
-  # WINDOWS WORKER (win server 2019)
+  ############ Windows worker node #1 ############
   config.vm.define :winw1 do |winw1|
-    winw1.vm.host_name = "winw1"
-    winw1.vm.box = "sig-windows-dev-tools/windows-2019"
+    winw1.vm.box = "mloskot/sig-windows-dev-tools-windows-2019"
     winw1.vm.box_version = "1.0"
+    winw1.vm.communicator = "winrm"
```

Comment:
ok , so we still use winrm ? just wanna clarify ...

Reply:
Yes, but I [have forced its startup](https://github.com/mloskot/sig-windows-dev-tools-boxes/blob/main/README.md#boxes) a bit. I have not tried to fiddle with SSH.